tests/plugins/imagebase/features/linkcommand.js

```diff
@@ -0,0 +1,59 @@
+/* bender-tags: editor,widget */
+/* bender-ckeditor-plugins: imagebase,toolbar, link,contextmenu */
+
+( function() {
+	'use strict';
+
+	bender.editors = {
+		classic: {},
+
+		divarea: {
+			config: {
+				extraPlugins: 'divarea'
+			}
+		},
+
+		inline: {
+			creator: 'inline'
+		}
+	};
+
+
+	var widgetHtml = '<figure><img src="%BASE_PATH%_assets/logo.png"></figure>',
+		tests = {
+			tearDown: function() {
+				var currentDialog = CKEDITOR.dialog.getCurrent();
+
+				if ( currentDialog ) {
+					currentDialog.hide();
+				}
+			},
+			setUp: function() {
+				var plugin = CKEDITOR.plugins.imagebase,
+					editors = this.editors;
+				CKEDITOR.tools.array.forEach( CKEDITOR.tools.objectKeys( editors ), function( editor ) {
+					plugin.addImageWidget( editors[ editor ], 'testWidget', plugin.addFeature( editors[ editor ], 'link', {} ) );
+				} );
+			},
+
+			'test link option in context menu': function( editor, bot ) {
```

Comment:
There are missing tests for at least such cases:

* trying to edit link on widget that is already linked
* trying to unlink widget